website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/setup-azure.md

```diff
@@ -101,16 +100,186 @@ Once you connect your Azure AD app and Azure DevOps, you need to provide dbt Clo
 Your Azure AD app should now be added to your dbt Cloud Account. People on your team who want to develop in dbt Cloud's IDE can now personally [authorize Azure DevOps from their profiles](dbt-cloud/cloud-configuring-dbt-cloud/authenticate-azure).
 
 ## Connecting a service user
-Because Azure DevOps forces all authentication to be linked to a user's permissions, we recommend you create a "service user" to manage access to your dbt Cloud account. For dbt Cloud scheduled runs, you should avoid linking authentication to an individual Azure DevOps user. Primarily this prevents your dbt Cloud production runs from failing if the person leaves your organization causing a loss of read access for the dbt repository. 
+Because Azure DevOps forces all authentication to be linked to a user's permissions, we recommend you create a "service user" in Azure DevOps whose permissions which will be used to power headless actions in dbt Cloud such as deployment runs and CI. A service user is a pseudo user set up in the same way an admin would set up a real user, but it's given permissions specifically scoped for service to service interactions. You should avoid linking authentication to a real Azure DevOps user because if this person leaves your organization, dbt Cloud will lose privileges to the dbt Azure DevOps repositories, causing production runs to fail.
+
+### More on Service Users
+
+A service user account must have the following permissions for all Azure DevOps projects/repos you want accessible in dbt Cloud:
 
-A "service user" account is a pseudo user account in Azure DevOps with read access to all dbt repos for the whole dbt Cloud account. This account enables you to scope permissions appropriately and preserve access. 
+ - **Project Reader**
+ - **ViewSubscriptions**
+ - **EditSubscriptions**
+ - **DeleteSubscriptions** * 
```

Comment:
any idea why these permissions are needed?

Reply:
@dataders I see the reasons for these permissions below, so I added a sentence to your suggestion above!